tutorials/formatter/README.md

```diff
@@ -0,0 +1,59 @@
+##CsvFormatter Example
+
+This example shows how to use CsvFormatter to customize the delimiters and the schema string for formatting.  
```

Comment:
Unclear what we mean by " to customize the delimiters and the schema string for formatting" You may remoe this line as the next line tells what the operator does